src/main/java/seedu/duke/Ui.java

```diff
@@ -0,0 +1,72 @@
+package seedu.duke;
+
+import task.Task;
+
+import java.util.Scanner;
+
+public class Ui {
+
+    private final Scanner sc;
```

Comment:
Constant variables should be capitalised according to the coding standards. I noticed similar issues in other areas as well.